online-inference/bloom-175b/bloom-175b-inferenceservice.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: serving.kubeflow.org/v1beta1
+kind: InferenceService
+metadata:
+  name: bloom-175b
+spec:
+  predictor:
+    minReplicas: 1
+    maxReplicas: 1 
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+              - key: gpu.nvidia.com/class
+                operator: In
+                values:
+                  - A40
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 10
+          preference:
+            matchExpressions:
+              - key: topology.kubernetes.io/region
+                operator: In
+                values:
+                  - LAS1
+    containers:
+      - name: kserve-container
+        image: tweldoncw/bloom-175b:a3c6d36
+        command:
+          - "python3"
+          - "/workspace/bloom.py"
+        env:
+          - name: MODEL_NAME
+            value: 'bigscience/bloom'
+          - name: STORAGE_URI # Kserve mounts the PVC at /mnt/pvc/
+            value: pvc://bloom-175b-model-cache/
+          - name: TRANSFORMERS_CACHE
+            value: /mnt/pvc/
+          - name: MIN_LENGTH
```

Comment:
You can note with a comment that these are defaults and can be overridden in the request